### PR TITLE
feat: ConfigProvider static function support `iconPrefixCls`

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    ConfigProvider static function `config` support `iconPrefixCls`.       |
| 🇨🇳 Chinese |    ConfigProvider 静态方法 `config` 支持配置 `iconPrefixCls`。      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
等 https://github.com/ant-design/ant-design/pull/30926 然后 rebase 一下。

**Review by crazyair — COMMENTED** [review]
LGTM

File: components/notification/index.tsx [hunk 1]
```diff
@@ -221,15 +227,17 @@ function getRCNoticeProps(args: ArgsProps, prefixCls: string) {
 
   return {
     content: (
-      <div className={iconNode ? `${prefixCls}-with-icon` : ''} role="alert">
-        {iconNode}
-        <div className={`${prefixCls}-message`}>
-          {autoMarginTag}
-          {message}
+      <ConfigProvider iconPrefixCls={iconPrefixCls}>
```
[the remark this answers is not in the document]
Comment (zombieJ) [reply components/notification/index.tsx:230]: 嗯，我调整了一下顺序，因为 Icon 不仅仅是 content 里有，外部也会有。

**Review by crazyair — APPROVED** [review]
LGTM

**Pull request merged**